ci-operator/step-registry/rhcs/e2e/setup/rhcs-e2e-setup-ref.yaml

```diff
@@ -29,6 +29,8 @@ ref:
     default: ""
   - name: "RHCS_VERSION" #if setup a job with RHCS_SOURCE the source in all tf manifest will be updated
     default: ""
+  - name: "ENABLE_SHARED_VPC"
```

Comment:
Will this env variable overwrite the configuration in profile?

Reply:
No. It's for Prow job only. 

The configuration for profile is `shared_vpc: true` https://github.com/terraform-redhat/terraform-provider-rhcs/pull/569/files#diff-1afc179e83a78ce9bcfc4e2868861c93c2f089d81345e0c7a0d4944d80d93a40R100